frontend/www/js/omegaup/components/contest/NewForm.vue

```diff
@@ -256,32 +264,50 @@ import DateTimePicker from '../DateTimePicker.vue';
 export default class NewForm extends Vue {
   @Prop() data!: omegaup.Contest;
   @Prop() update!: boolean;
+  @Prop() allLanguages!: string[];
+  @Prop() initialStartTime!: Date;
+  @Prop() initialFinishTime!: Date;
 
   T = T;
-  alias = this.data.alias;
-  availableLanguages = this.data.available_languages;
-  contest = this.data;
-  contestantMustRegister = this.data.contestant_must_register;
-  description = this.data.description;
-  feedback = this.data.feedback;
-  finishTime = this.data.finish_time;
-  scoreboard = this.data.scoreboard;
-  languages = this.data.languages;
-  needsBasicInformation = this.data.needs_basic_information;
-  penalty = this.data.penalty;
-  penaltyType = this.data.penalty_type;
-  penaltyCalcPolicy = this.data.penalty_calc_policy;
-  pointsDecayFactor = this.data.points_decay_factor;
-  requestsUserInformation = this.data.requests_user_information;
-  startTime = this.data.start_time;
-  showPenalty = this.data.show_penalty;
-  showScoreboardAfter = this.data.show_scoreboard_after;
-  submissionsGap = this.data.submissions_gap;
-  title = this.data.title;
+  alias = this.data ? this.data.alias : null;
```

Comment:
uy, esto te va a gustar:

```suggestion
  alias = this?.data ?? null;
```

a partir de [TypeScript 3.7](https://medium.com/@fuchodeveloper/optional-chaining-and-nullish-coalescing-in-typescript-7c5ea1c838b) se introdujeron [`?.`](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining) y [`??`](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#nullish-coalescing).

Reply:
Excelente, ya realicé el cambio